packages/web-runtime/src/components/UploadInfo.vue

```diff
@@ -226,9 +226,9 @@ export default defineComponent({
         return this.$gettext('Upload cancelled')
       }
       if (Object.keys(this.errors).length) {
-        const isLocked = Object.values(this.errors).some((error: any) => error.statusCode === 423)
+        const isLocked = Object.values(this.errors).some((error: any) => error?.statusCode === 423)
         if (isLocked) {
-          return this.$gettext('Upload failed because the current folder is locked.')
+          return this.$gettext('Upload failed: Locked')
```

Comment:
I'd say it's sufficient to have the detailed error message down below, no need to handle the lock information here as well.

Reply:
Got it